.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,7 @@
+---
+name: "❓ Questions"
+about: Please use https://forum.serverless.com, StackOverflow or other forums for questions
+---
+
+Please use https://forum.serverless.com, StackOverflow or other forums for questions.
+This issue tracker is dedicated for bug reports and feature requests.
```

Comment:
Maybe we should add a line that we'll close question issues immediately.

Reply:
Added in https://github.com/serverless/serverless/pull/6786/commits/610f93d0739030d50b4f80d3bcff787ce35775e2